src/app/views/App.tsx

```diff
@@ -286,6 +286,14 @@ class App extends Component<IAppProps, IAppState> {
     this.props.actions!.toggleSidebar(properties);
   }
 
+  private closeDialog = (): void => {
+    this.setState({ hideDialog: true });
+  };
+
+  private showDialog = (): void => {
+    this.setState({ hideDialog: false });
+  };
+
```

Comment:
Note: this is the same style as what was written in History.tsx so I followed the same coding formate